mypy/checkexpr.py

```diff
@@ -2211,6 +2213,13 @@ def accept(self,
         self.type_context.pop()
         assert typ is not None
         self.chk.store_type(node, typ)
+
+        if (disallow_any and
+                not self.chk.is_stub and
+                has_any_type(typ) and
+                'expr' in self.chk.options.disallow_any):
```

Comment:
Yeah, I think it's easier when you're looking through the code. 
Also, it might be a little faster.